d3/js/AliTV.js

```diff
@@ -959,6 +952,217 @@ AliTV.prototype.getOuterRadius = function() {
  * @author Sonja Hohlfeld
  */
 AliTV.prototype.getGenomeDistance = function() {
-	var genomeDistance = this.getCanvasHeight() * 1 / this.filters.karyo.genome_order.length;
-	return genomeDistance;
+	var genomeDistance = (this.getCanvasHeight() - this.getKaryoHeight()) / (this.filters.karyo.genome_order.length - 1);
+	return Math.round(genomeDistance);
+};
+
+/**
+ * This method should call other filter functions in order to filter the visible chromosomes.
+ * @returns visibleChromosomes: returns only chromosomes which are visible
+ * @author Sonja Hohlfeld
+ */
+AliTV.prototype.filterChromosomes = function() {
+	var visibleChromosomes = this.data.karyo.chromosomes;
+	if (this.filters.showAllChromosomes === false) {
```

Comment:
you should add
```{javascript}
else{
    return visibleChromosomes;
}
```
to this if-clause. This way it is enforced that all other filters are ignored if the master switch ```this.filters.showAllChromosomes``` is set to ```true```.

In the html file this should be reflected by disabling all other filters if ```showAllChromosomes``` is ```true```. The opposite direction is not necessary. But the ```showAllChromosomes``` checkbox should be the first (above all other filters).